youtube_dl/extractor/peertube.py

```diff
@@ -528,12 +562,17 @@ def account_data(field):
         return {
             'id': video_id,
             'title': title,
-            'description': video.get('description'),
+            'description': description,
             'thumbnail': urljoin(url, video.get('thumbnailPath')),
             'timestamp': unified_timestamp(video.get('publishedAt')),
             'uploader': account_data('displayName'),
-            'uploader_id': account_data('uuid'),
-            'uploder_url': account_data('url'),
+            'uploader_id': '%s@%s' % (account_data('name'), account_data('host')),
+            'uploader_url': account_data('url'),
+            'channel': channel_data('displayName'),
+            'channel_id': '%s@%s' % (channel_data('name'), channel_data('host')),
```

Comment:
Similarly, that's not an id. Also breaks on `None`s.

Reply:
Keeping the two fields consistent until the above discussion on id is resolved.
- The same nickname can possibly refer to different accounts on different hosts with the same username. Therefore nickname in isolation is not an id.
- The same account/channel will have different numeric ids when accessed from different instances. This _could_ be used, but to me it seems like a poor identifier. This PeerTube extractor could then extract videos from multiple accounts with the same id, or the same account with different ids.
- `username@host` is certainly an id string, and the most robust id of the three. I do not see how it is problematic.

Fixing breaks on None.

Reply:
I've already pointed out: this is not a global unique identifier and it's not supposed to be.